python/ml4ir/base/model/relevance_model.py

```diff
@@ -207,6 +208,7 @@ def fit(
         monitor_metric: str = "",
         monitor_mode: str = "",
         patience=2,
+        use_wandb_tracking=False,
```

Comment:
Need to update the Args doc accordingly.

Reply:
(and precise what we mean by "tracking")

Reply:
I've updated this to mean experiment tracking everywhere.